integration_tests/tests/temporary-accommodation/manage/bedspace.cy.ts

```diff
@@ -487,21 +492,11 @@ context('Bedspace', () => {
 
     it('should show the existing bedspace information on the edit page', () => {
       // And there is an online premises in the database with an online bedspace
-      const characteristics: Array<Characteristic> = [
-        {
-          id: '12e2e689-b3fb-469d-baec-2fb68e15e85b',
-          name: 'Single bed',
-          serviceScope: 'temporary-accommodation',
-          modelScope: 'room',
-        },
-        {
-          id: '7dd3bac5-3d1c-4acb-b110-b1614b2c95d8',
-          name: 'Shared kitchen',
-          serviceScope: 'temporary-accommodation',
-          modelScope: 'room',
-        },
+      const bedspaceCharacteristics: Array<Cas3BedspaceCharacteristic> = [
+        cas3BedspaceCharacteristicsFactory.byDescription('Ground floor level access'),
+        cas3BedspaceCharacteristicsFactory.byDescription('Shared kitchen'),
```

Comment:
The factory for characteristics now returns more realistic characteristics based on the ones available on preprod at the moment: 'Single bed' didn't actually exist for bedspaces, so it's been swapped with another that does exist.